<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoadPic</class>
 <widget class="QWidget" name="LoadPic">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>644</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>575</width>
    <height>644</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>575</width>
    <height>644</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Исходное изображение</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(141, 196, 145);</string>
  </property>
  <widget class="QLabel" name="title_lbl">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>451</width>
     <height>81</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Book Antiqua</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Исходное изображение</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="loadpic_lbl">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>512</width>
     <height>512</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
